src/state/README.md

```diff
@@ -1,14 +1,36 @@
 # State Manager
+The `StateManager` is a class that manages a group of subscribable [`Property`](./Property.js) instances.
+The provides functionality for registration of new properties, and for subscribing to existing properties.
 
-A class for managing a group of subscribable properties together. Allows for the registration of new properties.
+An example usage:
 
-For example:
-```
+```javascript
 var manager = new StateManager();
+
+// adds a new field called 'paused' to the manager, and sets it initially to false
```

Comment:
// adds a new field called 'paused' to the manager with an initial value of false